BUILD.bazel

```diff
@@ -330,6 +344,21 @@ cc_binary(
     ],
 )
 
+cc_binary(
+    name = "spirv-lint",
+    srcs = [
+        "tools/io.h",
```

Comment:
This should be in `hdrs`.  But the file does not seem to be used by `lint.cpp`.  Remove?

Reply:
It'll be used when reading in the input file, but I removed it for now.